packages/wrangler/src/user.tsx

```diff
@@ -412,7 +412,12 @@ export function reinitialiseAuthTokens(config?: UserAuthConfig): void {
   LocalState = getAuthTokens(config);
 }
 
-export function getAPIToken(): string | undefined {
+export type TokenWithType = {
+  type: "API" | "OAuth";
+  token: string | undefined;
+};
+
+export function getAPIToken(): TokenWithType {
```

Comment:
Don't we want the following?

```suggestion
export type TokenWithType = {
  type: "API" | "OAuth";
  token: string;
};

export function getAPIToken(): TokenWithType | undefined {
```